### PR TITLE
Code formatting homework StoyanElenkov

### PR DESCRIPTION
**Pull request by stoyanelenkov**
Code formatting homework Stoyan

### CODE REVIEW

File: formatting-code/stoyanelenkov/java_script_code.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    addScroll = true;
+}
+
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+document.onmousemove = mouseMove;
+
+if ( b == "Netscape" ) {
```
Comment (victorlazov) [inline formatting-code/stoyanelenkov/java_script_code.js:15]: Spaces between brackets and conditions.

File: formatting-code/stoyanelenkov/java_script_code.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+function mouseMove( evn ) {
+    if ( b == "Netscape" ) {
+        pX = evn.pageX - 5;
+        pY = evn.pageY;
+    } else {
+        pX = event.x - 5;
+        pY = event.y;
+    }
```
Comment (victorlazov) [inline formatting-code/stoyanelenkov/java_script_code.js:26]: New line.

File: formatting-code/stoyanelenkov/java_script_code.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+function showMenu2() {
+    if ( b == "Netscape" ) {
+        theLayer = eval( 'document.layers[\'menu2\']' );
+        theLayer.visibility = 'show';
+    } else {
+        theLayer = eval( 'document.all[\'menu2\']' );
+        theLayer.style.visibility = 'visible';
+    }
+} // fostata
```
Comment (victorlazov) [inline formatting-code/stoyanelenkov/java_script_code.js:116]: Is this comment appropriate?

File: formatting-code/stoyanelenkov/java_script_code.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        theLayer = eval( 'document.layers[\'ToolTip\']' );
+
+        if ( ( pX + 120 ) > window.innerWidth) {
+            pX = window.innerWidth - 150;
+        }
+
+        theLayer.left = pX + 10;
+        theLayer.top = pY + 15;
+        theLayer.visibility = 'show';
+    } else {
```
Comment (asparuh-krastev) [inline formatting-code/stoyanelenkov/java_script_code.js:49]: Stick to:
}
else {

File: formatting-code/stoyanelenkov/java_script_code.js [hunk 5]
```diff
@@ -0,0 +1,116 @@
+b = navigator.appName;
+addScroll = false;
+
+if ( ( navigator.userAgent.indexOf ( 'MSIE 5' ) > 0) ||
+    ( navigator.userAgent.indexOf ( 'MSIE 6' ) ) > 0) {
```
Comment (akraychev) [inline formatting-code/stoyanelenkov/java_script_code.js:5]: I think ) { should go on next line

File: formatting-code/stoyanelenkov/java_script_code.js [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+if ( b == "Netscape" ) {
+    document.captureEvents( Event.MOUSEMOVE );
+}
+
+function mouseMove( evn ) {
+    if ( b == "Netscape" ) {
+        pX = evn.pageX - 5;
+        pY = evn.pageY;
+    } else {
```
Comment (akraychev) [inline formatting-code/stoyanelenkov/java_script_code.js:23]: else { should be on next line

**Pull request merged**